openedx/core/djangoapps/user_api/accounts/views.py

```diff
@@ -398,14 +402,17 @@ def post(self, request):
             with transaction.atomic():
                 # 1. Unlink LMS social auth accounts
                 UserSocialAuth.objects.filter(user_id=request.user.id).delete()
-                # 2. Change LMS password & email
+                # 2. Retire any objects linked to the user via their email
+                CourseEnrollmentAllowed.delete_by_user_value(request.user.email, field='email')
```

Comment:
I thought we agree we don't have these two lines in this API endpoint? I thought we said, the original user emails can be retrieved from the retirementStatus model?

Reply:
ah, right, I just removed the 2 P2 calls, still have to do this.